scripts/script.py

```diff
@@ -2,7 +2,36 @@
 import asyncio
 import json
 from datasets import Dataset
+from utils import transform_to_jsonl
 import jsonlines
+from argparse import ArgumentParser
+
+
+def argument_parser():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--input_file",
+        type=str,
+        default="word.txt",
+        help="Input file containing the list of words to fetch",
+    )
+    parser.add_argument(
+        "--output_file",
+        type=str,
+        default="../data/dataset.jsonl",
+        help="Output file to store the fetched data",
+    )
+    parser.add_argument(
+        "--delay", type=int, default=1, help="Delay between each request in seconds"
+    )
+    parser.add_argument(
+        "--push_to_hub", type=bool, default=True, help="Push the dataset to the hub"
+    )
+    parser.add_argument("--token", type=str, default="", help="Hugging Face API token")
```

Comment:
suggestion
```python
parser.add_argument("--token", type=str, default=os.getenv("HUGGINGFACE_TOKEN", ""), help="Hugging Face API token (optionnel)")
```

Reply:
@dofbi For sure, I think we can improve it in the future! For now, these minor changes should do the job ! Thanks for the improvement made it

Reply:
Hello @dofbi  can i close this PR ?

Reply:
Hello @abdouaziz 
J'attendais tes derniers changements pour faire le merge
Désolé pour l'incompréhension 🙏🏾